test/behaviours/clear-feedback.spec.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+
+const mockConfig = {
+  notify: {
+    feedbackEmailSessionName: 'feedback-email'
+  }
+};
+
+const Behaviour = proxyquire('../../apps/feedback/behaviours/clear-feedback', {'../../../config': mockConfig});
+
+
+describe('Clear feedback behaviour', () => {
+
+  let req;
+  let res;
+  let ClearFeedback;
+  let testInstance;
+  let nextStub;
+
+  beforeEach(() => {
+    req = {
+      sessionModel: {
+        get: sinon.stub(),
+        set: sinon.stub(),
+        unset: sinon.stub()
+      },
+      get: sinon.stub()
+    };
+
+    res = {
+      redirect: sinon.stub()
+    };
+  });
+
+  describe('getValues', () => {
+
+    class Base {
+      getValues() {}
+    }
+
+    beforeEach(() => {
+      ClearFeedback = Behaviour(Base);
+      testInstance = new ClearFeedback();
+      nextStub = sinon.stub();
+    });
+
+    it('should call un set all of the feedback variables on req after submission', () => {
```

Comment:
it should unset feedback form fields and feedback email reference on the session